lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -1088,6 +1088,15 @@ MStatus MtohRenderOverride::Render(
        _displayStyleSceneIndex->SetRefineLevel({true, delegateParams.refineLevel});
     }
 
+    // Toggle textures in the material network
+    const unsigned int currentTextureMode = drawContext.getDisplayStyle();
```

Comment:
I think the variable is wrongly named here. It's not the currentTextureMode but the currentDisplayStyle which contains a bit about the texture mode. Could you please renamed it ?